install/linux/docker-ce/ubuntu.md

```diff
@@ -54,7 +54,8 @@ networks, are preserved. The Docker CE package is now called `docker-ce`.
 
 ### Supported storage drivers
 
-Docker CE on Ubuntu supports `overlay2` and `aufs` storage drivers.
+Docker CE on Ubuntu supports `overlay2`, `aufs` and `btrfs` storage drivers.
+Please note, however, that in Docker EE `btrfs` is only supported on SLES (see [btrfs](/engine/userguide/storagedriver/btrfs-driver.md)).
```

Comment:
This article is for Docker CE running on Ubuntu, so I don't think we should talk about Docker EE here. You can find the docs for EE here:
* [Ubuntu](https://docs.docker.com/install/linux/docker-ee/ubuntu/)
* [SLES](https://docs.docker.com/install/linux/docker-ee/suse/)